src/txdb.cpp

```diff
@@ -386,7 +386,7 @@ bool CCoinsViewDB::Upgrade() {
             if (count++ % 256 == 0) {
                 uint32_t high = 0x100 * *key.second.begin() + *(key.second.begin() + 1);
                 int percentageDone = (int)(high * 100.0 / 65536.0 + 0.5);
-                uiInterface.ShowProgress(_("Upgrading UTXO database") + "\n"+ _("(press q to shutdown and continue later)") + "\n", percentageDone);
+                uiInterface.ShowProgress(_("Upgrading UTXO database") + "\n"+ _("(press q to shutdown and continue later)") + "\n", percentageDone, std::function<void(void)>());
```

Comment:
In commit "Add cancel function reference to ShowProgress"

No need to write `std::function<void(void)>()` here and 12 other places. You can just pass {} or nullptr.

Reply:
Oh, I didn't realize that that would work. Done.